docs/CONTRIBUTING.md

```diff
@@ -21,34 +18,21 @@ These contributions come in two forms: issues and pull requests.
 
 ## Issues
 
-If you spot anything
-that conflicts with our values,
-opening a GitHub Issue is a great way
-to give us specific feedback.
+If you spot anything that conflicts with our values, opening a GitHub Issue is a great way to give us specific feedback.
 
-To make an issue,
-refer to the [GitHub Issues Workflow](https://github.com/circleci/circleci-docs/wiki/GitHub-Issues-Workflow) wiki page.
+To make an issue, refer to the [GitHub Issues Workflow](https://github.com/circleci/circleci-docs/wiki/GitHub-Issues-Workflow) wiki page.
```

Comment:
Maybe change "make an issue" to "create an issue"? I'd also maybe make this more concise and add it with the above sentence. 

> If you spot anything that conflicts with our values, opening a GitHub Issue is a great way to give us specific feedback. Refer to the [GitHub Issues Workflow](https://github.com/circleci/circleci-docs/wiki/GitHub-Issues-Workflow) wiki page to create an issue.